theia-extensions/viewer-prototype/src/browser/trace-viewer/trace-viewer-toolbar-commands.ts

```diff
@@ -0,0 +1,22 @@
+import { Command } from '@theia/core';
+
+export namespace TraceViewerToolbarCommands {
+
+    export const ZOOM_IN: Command = {
+        id: 'trace.viewer.zoomin',
+        label: 'Zoom In',
+        iconClass: 'fa fa-plus',
```

Comment:
https://fontawesome.com/v4.7/icon/search-plus ?

Reply:
https://fontawesome.com/v4.7/icon/plus-square-o !!! this one! this one!

Reply:
![image](https://user-images.githubusercontent.com/70230232/122905704-0fef9480-d317-11eb-95e2-d5848355c45c.png)
 
They're very dull for some reason :/